akka-actor-typed-tests/src/test/java/akka/actor/typed/javadsl/ActorCompile.java

```diff
@@ -84,12 +83,12 @@ public MyMsgB(String greeting) {
   static class MyBehavior extends ExtensibleBehavior<MyMsg> {
 
     @Override
-    public Behavior<MyMsg> receiveSignal(ActorContext<MyMsg> ctx, Signal msg) throws Exception {
+    public Behavior<MyMsg> receiveSignal(akka.actor.typed.ActorContext<MyMsg> ctx, Signal msg) throws Exception {
```

Comment:
`ExtensibleBehavior` ended up creating the same `ActorContext` naming conflict as `BehaviorInterceptor`.

This makes me think it might come up again and again, with each adapter to `javadsl`/`scaladsl` creating the need for more new ones. I worked around it for now pending further discussion.

Reply:
Perhaps we should rename that common `ActorContext` type? I have always though that it was one too many of them. Should be rarely exposed. Both `ExtensibleBehavior` and `BehaviorInterceptor` are power user tools, typically only for library authors.